deps/Makefile

```diff
@@ -414,7 +414,7 @@ LLVM_FLAGS += --enable-libcpp
 endif # USE_LIBCPP
 ifeq ($(OS), WINNT)
 LLVM_FLAGS += --with-extra-ld-options="-Wl,--stack,8388608" LDFLAGS=""
-LLVM_CPPFLAGS += -D__USING_SJLJ_EXCEPTIONS__ -D__CRT__NO_INLINE
+LLVM_CPPFLAGS += -D__USING_SJLJ_EXCEPTIONS__ -D__CRT__NO_INLINE -DMINGW_HAS_SECURE_API=1
```

Comment:
Interestingly enough, cygwin will kill your ssh session, and continue the update.  I then restart the server through Horizon, and it seems to work just fine.  A cleaner solution is to login over RDP and do everything in cmd.exe of course, but that's a much bigger hassle because it requires you to have the auto-generated windows password written down, etc...